tests/distributed/test_dist_graph_store.py

```diff
@@ -730,7 +730,8 @@ def check_dist_graph_hetero(g, num_clients, num_nodes, num_edges):
 
     # Test reading edge data
     eids = F.arange(0, int(g.number_of_edges("r1") / 2))
-    feats1 = g.edges["r1"].data["feat"][eids]
+    c_etype = g.to_canonical_etype("r1")
+    feats1 = g.edges[c_etype].data["feat"][eids]
```

Comment:
Is this PR a breaking change? If not, we shouldn't change the unit test but add a new test instead.

Reply:
It's not a breaking change. The change I made for existing test is add more tests to cover the change here, not identical to modifying existing one. In specific here, access via `"r1"` is tested in other part in this testcase, so I think it's ok to change the access way here to cover the access via canonical etype.

Reply:
accessing via etype and canonical etype are both added.